### ISSUE TITLE
update Ribbon AGR slim to be compatible 

### ISSUE THREAD

**Issue by nathandunn** [geneontology/ribbon#56]
TODO

- [x] make sure the slims line up: https://github.com/geneontology/ribbon/blob/master/src/data/agr.js
- [x] fix just the CC portion: https://github.com/geneontology/ribbon/blob/master/src/data/expression.js 
- [x] create agr_ui PR

via reference for agr_slim: http://snapshot.geneontology.org/ontology/go.obo

fixes here: https://agr-jira.atlassian.net/browse/AGR-1163 .

**Comment by lpalbou** [comment]
Just an update on that: I have talked with Chris and we will simplify the BioLink API call on the API side (e.g. /slimmer/function?subset=agr_slim&subject=xxx), hence the files agr.js and I believe expression.js would be deprecated in favor of this API call.  It would also ensure that a slim is always the same wherever you use it.

There will also be a /ontology/slim/{id} to retrieve metadata on a specific slim, should you need it.

**Comment by nathandunn** [comment]
:+1

We are still allowing folks to pass in a slim, though, right?

Nathan

**Comment by sibyl229** [comment]
Thanks @lpalbou . I should be fine I think. I'm fetching data from WormBase database.

**Comment by lpalbou** [comment]
As the PR on BioLink / Ontobio will take some time, I have corrected the agr.js file manually and done this intermediary PR: https://github.com/geneontology/ribbon/pull/65

@selewis was right, there is an obsolete term, GO:0000988 (obsolete transcription factor activity, protein binding). I am unsure how the slimmer will work on an obsolete term so this should be fixed. Following comments on [jira](
https://agr-jira.atlassian.net/browse/AGR-1163), I have removed it from the slim at the moment.

Note: this AGR slim is indeed larger, we'll have to see how it renders on AGR
